DataFormats/L1Trigger/src/classes_def.xml

```diff
@@ -261,4 +261,5 @@
   <class name="std::vector<l1t::L1DataEmulResult>"/>
   <class name="edm::Wrapper<l1t::L1DataEmulResultBxCollection>"/>
 
+  <class name="std::pair<edm::Ptr<l1t::L1Candidate>,float>"/>
```

Comment:
@silviodonato Note that the error complains about the dictionary of `edm::Ptr<l1t::L1Candidate>` being missing.

I'm a bit puzzled by the error though, I would have expected a missing dictionary to show up with a different kind of exception.